test/byContract/IexecPoco/IexecPoco1.test.ts

```diff
@@ -94,6 +98,18 @@ describe('IexecPoco1', () => {
             dataset: datasetPrice,
             workerpool: workerpoolPrice,
         };
+        ({ orders } = buildOrders({
+            assets: ordersAssets,
+            prices: ordersPrices,
+            requester: requester.address,
+            beneficiary: beneficiary.address,
+            tag: standardDealTag,
```

Comment:
I remember you were ok to switch set TEE as default :thinking: 
```suggestion
            tag: teeDealTag,
```